### PR TITLE
submition

### PR DESCRIPTION
**Pull request by tsolman**

### CODE REVIEW

**Comment by tchap** [comment]
Hi @tsolman , I am going to review your PR. I am wondering, why is the code commit made by a different GitHub account, @kon-flash ?

**Comment by tchap** [comment]
Also you managed to commit the resulting Go binary, you might want to remove it from the commit and force-push so that we don't have to clone megabytes of data. No problem, can happen to anyone...

**Comment by tsolman** [comment]
I used my other laptop to push the code and I was logged in with my other profile. ( Belongs to current company)

File: main.go [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+		},
+	}
+
+	users = append(users, User{ID: "1", Name: "Tsolakidis",
+		Asset: initassets,
+	})
+	assets = initassets
+
+	r.HandleFunc("/api/health", health).Methods("GET")
+	r.HandleFunc("/api/user/", createUser).Methods("POST")
```
Comment (lmacko) [inline main.go]: What would be the response for a POST request to `/api/user` with an invalid data? e.g.: `{"something":invalid}` 
How would you harden the current solution?

Comment (tsolman) [reply main.go]: Well for a production ready solution, i would have a validation function that would check
i.e. if name is null or undefined and returns the appropriate error.
OR use a golang module.
The responce would look like `{"validation_error": "invalid request data"}` or `{"validation_error": "Name field is required"} `

**Pull request closed without merging**